src/node_report.cc

```diff
@@ -1030,14 +1039,20 @@ static void PrintResourceUsage(std::ostream& out) {
 #if defined(__APPLE__) || defined(_AIX)
     snprintf( buf, sizeof(buf), "%ld.%06d", stats.ru_utime.tv_sec, stats.ru_utime.tv_usec);
     out << "\n  User mode CPU: " << buf << " secs";
+    cpu_abs = std::stod(buf);
     snprintf( buf, sizeof(buf), "%ld.%06d", stats.ru_stime.tv_sec, stats.ru_stime.tv_usec);
     out << "\n  Kernel mode CPU: " << buf << " secs";
+    cpu_abs += std::stod(buf);
```

Comment:
Parsing back our floating point numbers seems like more work than neccessary. Could you just use arithmetic to add up `stats.ru_utime.tv_sec, stats.ru_utime.tv_usec, stats.ru_stime.tv_sec and stats.ru_stime.tv_usec` instead?
I think you can do it outside the `#if defined` block as well then so there's less duplication.